providers/inline-evaluating-provider/test/java/dev/openfeature/contrib/providers/inlineeval/ExampleFetcher.java

```diff
@@ -0,0 +1,23 @@
+package dev.openfeature.contrib.providers.inlineeval;
+
+import dev.openfeature.sdk.EvaluationContext;
+
+public class ExampleFetcher implements RuleFetcher {
+
+    @Override
+    public String getRuleForKey(String key) {
+        // In a real version of this, these rules should be cached.
+        return """
+                {"if": [
+                  {"or": [
+                    {"in": [{"var": "userId"}, [1,2,3,4]]},
+                    {"in": [{"var": "category"}, ["pies", "cakes"]]},
+                  ]},
+                  true,
+                  false
+                ]}
+                """;
```

Comment:
I'm surprised `TextBlocks` work in the CI TBH, since we use JDK8 there and they seem to be added with Java 15.